addons/account/static/src/xml/legacy_tax_totals.xml

```diff
@@ -44,7 +44,7 @@
                 <t t-foreach="totals.value.subtotals" t-as="subtotal" t-key="subtotal['name']">
                     <tr>
                         <td class="o_td_label">
-                            <label class="o_form_label o_tax_total_label" t-esc="subtotal['name']"/>
+<!--                            <label class="o_form_label o_tax_total_label" t-esc="subtotal['name']"/>-->
```

Comment:
???

Reply:
wrongly committed and it has been revered